bin/qtui-rebind

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+"""
+    Simple script to rewrite Qt ui files to change shortcut bindings.
```

Comment:
Agreed. But as I understand this discussion it is about exchanging ideas rather than truly convincing someone of their wrong ;)

And to be picky there aren't any other module level docstrings.